phoenix-core/src/main/java/org/apache/phoenix/execute/ClientAggregatePlan.java

```diff
@@ -87,6 +89,25 @@ public ClientAggregatePlan(StatementContext context, FilterableStatement stateme
     }
 
     @Override
+    public Cost getCost() throws SQLException {
+        Long byteCount = getEstimatedBytesToScan();
+        if (byteCount == null) {
+            return Cost.ZERO;
+        }
+
+        int parallelLevel = 1;
```

Comment:
So parallelLevel is 1 because aggregation is done client-side instead of pushed to the cluster, right? Would be good to doc the thinking behind calculating the cost throughout.